Protocol.md

```diff
@@ -235,4 +264,4 @@ Nadav Kohen <nadavk25@gmail.com>
 
 ![Creative Commons License](https://i.creativecommons.org/l/by/4.0/88x31.png "License CC-BY")
 <br>
-This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
\ No newline at end of file
+This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```

Comment:
What's this diff about?

Reply:
My editor just added a new line to the end